cmd/thanos/tools_bucket.go

```diff
@@ -14,6 +14,8 @@ import (
 	"text/template"
 	"time"
 
+	"github.com/thanos-io/thanos/pkg/ui/config"
```

Comment:
Minor nit. You should move this import down with other thanos imports.

Reply:
This is kind of an important rule that we respect.